now.json

```diff
@@ -13,6 +13,11 @@
     },
     {
       "handle": "filesystem"
+    },
+    {
+      "src": "\/(\S+?)\/.*",
```

Comment:
```suggestion
      "src": ""\\\/(\\S+?)\\\/.*"",
```
JSON escaping, no questions...  That's what happens if you try to code in github PR comments :-(

Reply:
Escaping the slash should not be necessary